scripts/dns-zone-and-server-certificates/create-or-update-all-test-dns-zones.sh

```diff
@@ -29,13 +30,17 @@ fi
 scriptDir=$( dirname "$0" )
 
 
-zoneProfile=$1
-parentZoneProfile=$2
+devZoneProfile=$1
+uatZoneProfile=$2
+parentZoneProfile=$3
 
 
-for envName in $( echo $environments )
+for envType in $( echo "dev uat" )
```

Comment:
Alla fine conviene fare una variabile envinronments com'era prima e fare il ciclo for, riorganizzando i parametri 
` echo "Usage: $0  <parent-zone-profile> <dev-zone-profile> [<uat-zone-profile>]....."`

Reply:
Ho riorganizzato i parametri e il ciclo for non l'ho mai tolto , ci avevo pensato a toglierlo ma poi veniva veramente brutto.